rapid7_insightidr/help.md

```diff
@@ -36,15 +36,111 @@ Example input:
 
 ### Actions
 
-#### Advanced Query
+#### Advanced Query on Log Set
 
-This action is used to run realtime queries into Insight IDR logs.
+This action is used to realtime query into an Insight IDR log set.
```

Comment:
Would be good to explain when to use Log vs Logset version of the action in each action's description. The names are very similar and a new IDR user may not know